src/sentry/models/projectownership.py

```diff
@@ -25,3 +26,59 @@ class Meta:
         db_table = 'sentry_projectownership'
 
     __repr__ = sane_repr('project_id', 'is_active')
+
+    def get_owners(self, event):
+        if self.schema is None:
+            return None
+
+        data = event.data
+
+        for rule in load_schema(self.schema):
+            if rule.test(data):
+                # This is O(n) to resolve, but should be fine for now
+                # since we don't even explain that you can use multiple
+                # let alone a number that would be potentially abusive.
+                owners = []
+                for o in rule.owners:
+                    try:
+                        owners.append(make_actor(o, self.project_id))
+                    except UnknownActor:
+                        continue
+                return owners
+
+        return None
+
+
+class UnknownActor(Exception):
+    pass
+
+
+def make_actor(owner, project_id):
+    """ Convert an Owner object into an Actor """
+    from sentry.api.fields.actor import Actor
+    from sentry.models import User, Team
+
+    if owner.type == 'user':
+        try:
+            user_id = User.objects.filter(
+                email__iexact=owner.identifier,
+                is_active=True,
+                sentry_orgmember_set__organizationmemberteam__team__projectteam__project_id=project_id,
```

Comment:
ORM god

The gist of this is "give me the user who matches this email address and who has access to this project through team and org membership relationships.